### PR TITLE
Kisti merge test

### PR DESCRIPTION
**Pull request by jpa198**

### CODE REVIEW

**Comment by jpa198** [comment]
This has a co-dependency on the related slurm_workflow PR

File: qcore/config.py [hunk 1]
```diff
@@ -13,30 +15,41 @@ def determine_machine_config(hostname=platform.node()):
         "maui"
     ):
         machine = "maui"
```
Comment (jasonmotha) [inline qcore/config.py:17]: Way too many strings. Would be good to enum the lot of them.

**Comment by jasonmotha** [comment]
Looks good. There's some good logical grouping going on.

**Comment by jasonmotha** [comment]
@jpa198 Cool breaking of master bro. 😭

**Comment by jpa198** [comment]
> @jpa198 Cool breaking of master bro. sob

See the below comment given last week...

> This has a co-dependency on the related slurm_workflow PR

**Pull request merged**